### PR TITLE
feat(error): localize firebase error

### PR DESCRIPTION
**Pull request by Oupsla**
Pr for https://github.com/convoo/login-fire/issues/111

I only add 2 translations when identifier is not linked to an account or when the password is wrong.
I also add the german translation (thanks to google trad, so I don't know if those are correct)

Feel free to add more translations or edit german translations !

### CODE REVIEW

File: login-fire-form.html [hunk 1]
```diff
@@ -365,7 +365,13 @@
...
         this.password = null;
       } else {
-        this.set('_lastError', e.detail.message);
+        // Localize firebase error if possible
+        var errorMessage = this.localize('lf-error-'+e.detail.code);
+        if(!errorMessage) {
+          this.set('_lastError', e.detail.message);
+        } else {
+          this.set('_lastError', errorMessage);
+        }
```
Comment (masonlouchart) [inline login-fire-form.html]: I guess we could replace the `if-else` block by `this.set('_lastError', errorMessage || e.detail.message);` but it's OK.

**Comment by masonlouchart** [comment]
Is there a German guy around here for validating the error messages translation? 😜

File: login-fire-form.html (hunk 1 above)
Comment (Oupsla) [reply login-fire-form.html]: Done :)

**Comment by masonlouchart** [comment]
I guess if there is a typo in any error massage developers will contribute.

**Pull request merged**